packages/analytics/analytics-chart/src/components/chart-plugins/ChartLegend.vue

```diff
@@ -66,44 +66,12 @@ const legendItemsRef = ref<HTMLElement[]>([])
 const showValues = inject('showLegendValues', true)
 const position = inject('legendPosition', ref(ChartLegendPosition.Right))
 const legendItemsTracker = ref<LegendItem[]>([])
-const legendHeight = ref<string>(KUI_SPACE_80)
-
-// Check if the legend wraps by comparing the top position of each item.
-const doesTheLegendWrap = () => {
```

Comment:
I think we still need some of this code, just playing around in the sandbox I see some issues with legend items overlapping 

![image](https://github.com/Kong/public-ui-components/assets/6026470/07af6d15-f3a2-499c-9011-063d95a1a24d)